Source/EasyNetQ.Serialization.NewtonsoftJson/EasyNetQ.Serialization.NewtonsoftJson.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>An adaptor to allow EasyNetQ to use Newtonsoft.Json serializer</Description>
+    <TargetFramework>netstandard2.0</TargetFramework>
+   <PackageTags>Newtonsoft;$(PackageTags)</PackageTags>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\EasyNetQ\Internals\ReadOnlyMemoryStream.cs" Link="ReadOnlyMemoryStream.cs" />
```

Comment:
If I understand correctly, then this file is convenient to use in various serializer packages without adding a dependency to the main package.